src/components/P/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+interface Props {
```

Comment:
Normally I'd make it more descriptive and use the name of the element as well. Here it is a bit tricky as it would be `PProps` 😁  I'd still do it to keep the convention, I think. 🤔